src/app/dataset-view/additional-components/metadata-component/components/form-components/input-field/input-field.component.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { FormControl, FormGroup, ReactiveFormsModule } from "@angular/forms";
+import { InputFieldComponent } from "./input-field.component";
+import { TooltipIconComponent } from "src/app/dataset-block/metadata-block/components/tooltip-icon/tooltip-icon.component";
+import { NgbTooltipModule } from "@ng-bootstrap/ng-bootstrap";
+
+describe("InputFieldComponent", () => {
+    let component: InputFieldComponent;
+    let fixture: ComponentFixture<InputFieldComponent>;
```

Comment:
Where are the tests for validations, errors displayed, ... ?

Reply:
added tests for validations